pkg/scheduler/scheduler.go

```diff
@@ -576,6 +594,13 @@ func (e entryOrdering) Less(i, j int) bool {
 	a := e.entries[i]
 	b := e.entries[j]
 
+	// Already has quota reserved
+	aHasQuota := workload.HasQuotaReservation(a.Obj)
+	bHasQuota := workload.HasQuotaReservation(b.Obj)
+	if aHasQuota != bHasQuota {
+		return aHasQuota
```

Comment:
Do we want to keep Workloads that require another scheduling pass at the top of the heap?

Reply:
actually, i don't think it matters for corretness, we will consider them anyway. Just wanted to have a simpler mental model, where they are "almost done", and have "top priority" to move forward with them asap. 

However, I don't think it changes actually the behavior much.

Reply:
Got it